ember-intl/addon-test-support/add-translations.ts

```diff
@@ -14,42 +12,49 @@ function pickLastLocale(localeName: string | string[]): string {
 }
 
 /**
- * Invokes the `addTranslations` method of the `intl` service. The first
- * parameter, the `localeName`, is optional and will default to the last
- * currently enabled locale. This means, that if you invoke this helper with
- * just translations, they will be added to the last locale and all other
- * locales will be tried before.
+ * Adds translations, as if the end-developer had somehow added them.
+ *
+ * The first parameter, `localeName`, is optional and defaults to
+ * the current locale (the last enabled locale). So if you invoke the
+ * test helper with just the translations, they will be added to the
+ * last locale and all other locales will be tried before.
```

Comment:
I didn't understand the last part, so I left the description as is. I found the documentation for testing also unclear, and will need to update its texts.